dependencies.props

```diff
@@ -47,17 +50,18 @@
     
     <XmlUpdateStep Include="CoreFx">
       <Path>$(MSBuildThisFileFullPath)</Path>
-      <ElementName>PlatformPackageVersion</ElementName>
-      <PackageId>Microsoft.NETCore.Platforms</PackageId>
+      <ElementName>CoreFxExpectedPrerelease</ElementName>
```

Comment:
This should just cause the "CoreFxExpectedPrerelease" property to be updated, instead of the "PlatformPackageVersions" property. The latter is now just based on the former.